packages/ui/src/theme/tokens/components/pagination.ts

```diff
@@ -63,6 +72,22 @@ export interface PaginationTokens {
   itemShared: PaginationItemSharedTokens;
 }
 
+// To be removed in the next major version release
+export const DUPLICATE_PAGINATION_STATES: ReplacementStateToken[] = [
```

Comment:
Based on my earlier comment, although it is less explicit, and maybe this is too much premature optimization, this could be a simple object where the key is tokenName and the value is path.

Reply:
+1 on just using an object instead of an array

Reply:
This has been converted to an tokenName:path object instead